maintainers/maintainer-list.nix

```diff
@@ -4028,6 +4028,11 @@
     keys = [{
       fingerprint = "1C4E F4FE 7F8E D8B7 1E88 CCDF BAB1 D15F B7B4 D4CE";
     }];
+  dgollings = {
+    email = "daniel.gollings+nixpkgs@gmail.com";
+    github = "dgollings";
+    githubId = 2032823;
+    name = "Daniel Gollings";
   };
```

Comment:
it seems like you accidentally deleted a bracket from the entry above :(

Reply:
gah :D, now?